cloudify/providerdeployment.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright (c) 2018 GigaSpaces Technologies Ltd. All rights reserved
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudify
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"fmt"
+	"os"
+)
+
+
+/*
+Example for deployments info Which need to be used by cloudify provider
+{
+   "deployments":[
+      {
+         "id":"dep-1",
+         "deployment_type":"node",
+         "node_data_data_type":"cloudify.nodes.Kubernetes.Node"
+      },
+      {
+         "id":"dep-2",
+         "deployment_type":"load",
+         "node_data_type":"cloudify.nodes.ApplicationServer.kubernetes.LoadBalancer"
+      }
+   ]
+}
+ */
+
+type DeploymentsInfo struct {
+	Deployments         []interface{}          `json:"deployments,omitempty"`
+}
+
+// Get deployments provider info needed to be used by kubernetes cloudify provider
+func ParseDeploymentFile(deploymentFile string) (*DeploymentsInfo, error) {
```

Comment:
Run make test, looks like it will show warnings about "wrong format of command, should be // **ParseDeploymentFile** ".

Reply:
@0lvin please check this https://github.com/cloudify-incubator/cloudify-rest-go-client/blob/testing/cloudify/providerdeployment.go this change is old and it is because of squash commits

Reply:
Can you elaborate more on this , what is the change needed for this ?

Reply:
````
golint ./src/github.com/cloudify-incubator/cloudify-rest-go-client/...
src/github.com/cloudify-incubator/cloudify-rest-go-client/cfy-go/info.go:279:9: if block ends with a return statement, so drop this else and outdent its block
src/github.com/cloudify-incubator/cloudify-rest-go-client/cloudify/providerdeployment.go:45:6: exported type DeploymentsInfo should have comment or be unexported
src/github.com/cloudify-incubator/cloudify-rest-go-client/cloudify/providerdeployment.go:49:1: comment on exported function ParseDeploymentFile should be of the form "ParseDeploymentFile ..."
````